zetta_utils/layer/volumetric/constant/backend.py

```diff
@@ -105,6 +105,9 @@ def get_chunk_size(self, resolution: Vec3D) -> Vec3D[int]:  # pragma: no cover
     def get_dataset_size(self, resolution: Vec3D) -> Vec3D[int]:  # pragma: no cover
         return Vec3D[int](0, 0, 0)
 
+    def get_bounds(self, resolution: Vec3D) -> VolumetricIndex:  # pragma: no cover
+        return VolumetricIndex.from_coords((0, 0, 0), (0, 0, 0), Vec3D[int](1, 1, 1))
```

Comment:
Not sure if it's important, but maybe it makes sense to make it virtually unbounded -- something life from min to max value of `int64`?

Reply:
Good idea - I'll include it in the next PR.

Reply:
Why not this one?